src/functions/asyncCredential/asyncCredentialHandler.ts

```diff
@@ -156,15 +151,15 @@ export async function lambdaHandler(
   }
 
   const recoverSessionService = dependencies.getRecoverSessionService(
-    sessionTableName,
-    sessionTableSubIndexName,
+    config.SESSION_TABLE_NAME,
```

Comment:
Is it worth deconstructing all these variables once after the config function? Suggestion only, feel free to dismiss

Reply:
🤷 if we want to. Will leave you to re-analyse this once you've finished all your dev. It adds extra lines of code